examples/ListBox-Sorted-StringList-model

```diff
@@ -1,5 +1,5 @@
 // Variation on ListBox model example to show use of StringList and StringSorter.
```

Comment:
That is not enough, you should look at other examples. You need a folder containing a main.rs file, then the folder should be added to the README.md and Cargo.toml; like other examples https://github.com/gtk-rs/gtk4-rs/blob/master/examples/Cargo.toml#L163

Reply:
Feel free to ping me by DM on Matrix and can guide through that.

Reply:
I'll do that tomorrow.  Way past my bed time.

Reply:
I think I have it figured out.  I created a codespace where I can make the changes.  Thanks for your help.